File: pinyougou_parent/pinyougou_manager_web/src/main/webapp/js/service/orderItemService.js
app.service('orderItemService', function ($http) {
    //旧的查询方法(弃置)								   
    this.findPage = function (pageNum, pageSize) {
        return $http.get('../orderItem/findPage.do?pageNum=' + pageNum + '&pageSize=' + pageSize)
    };

    //查询方法,读取列表数据绑定到表单中
    this.search = function (pageNum, pageSize, searchEntity) {
        return $http.post('../orderItem/findPage.do?pageNum=' + pageNum + "&pageSize=" + pageSize, searchEntity)
    };

    //删除方法
    this.removeOrderItemList = function (idList) {
        return $http.get('../orderItem/removeOrderItem.do?idList=' + idList)
    };

    //根据id查询
    this.findOne = function (id) {
        return $http.get('../orderItem/findOne.do?id=' + id)
    };

    //修改品牌
    this.updateOrderItem = function (orderItem) {
        return $http.post('../orderItem/updateOrderItem.do', orderItem)
    };

    //新增品牌
    this.saveOrderItem = function (orderItem) {
        return $http.post('../orderItem/saveOrderItem.do', orderItem)
    };   
});

File: pinyougou_parent/pinyougou_sellergoods_interface/src/main/java/com/pinyougou/sellergoods/service/OrderItemService.java
package com.pinyougou.sellergoods.service;
import java.util.List;
import com.pinyougou.pojo.TbOrderItem;							  
import entity.PageResult;
import entity.Result;
/**
 * orderItem服务层接口
 * @author jieweili
 *
 */
public interface OrderItemService {

    /**
     * 返回全部列表(已弃置)
     * @return
     */
    List<TbOrderItem> findAll();
 
    /**
     * 返回分页对象(过时)
     * @param pageNum
     * @param pageSize
     * @return
     */
	 
    PageResult findPage(Integer pageNum, Integer pageSize);

    /**
     * 添加数据
     * @param orderItem
     * @return
     */
    public Result saveOrderItem(TbOrderItem orderItem);

    /**
     * 根据id查询一个对象
     * @param id
     * @return
     */
    TbOrderItem getOrderItemById(Long id);

    /**
     * 修改对象
     * @param orderItem
     * @return
     */
    Result updateOrderItem(TbOrderItem orderItem);

    /**
     * 批量删除
     * @param idList
     * @return
     */
    Result removeOrderItem(Long[] idList);

    /**
     * 根据条件分页查询
     * @param orderItem
     * @param pageNum
     * @param pageSize
     * @return
     */
    PageResult listOrderItemByExample(TbOrderItem orderItem, Integer pageNum, Integer pageSize);
 
}
